src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,6 +74,26 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+        Player p = e.getPlayer();
+        if (Constants.MACHINE_BREAK_REQUIRES_WRENCH
+            && (!Items.WRENCH.isSimilar(p.getInventory().getItemInMainHand()))
```

Comment:
Never use `.isSimilar()`. Never...

Also use `e.getItemInHand()`, not only the main hand item.

Reply:
I was thinking about this, and I realized a potential bug could be a player holding a wrench in the offhand and not using it to 
 while breaking the block with a pickaxe or whatever. Also why i restricted the wrench to only be mainhand use